app/models/organization.rb

```diff
@@ -49,7 +49,7 @@ class Organization < ActiveRecord::Base
   include DivisionBased
 
   belongs_to :division
-  belongs_to :country
+  belongs_to :country, required: true
```

Comment:
"You can use :inverse_of to avoid an extra query during validation."
https://apidock.com/rails/ActiveRecord/Associations/ClassMethods/belongs_to

Reply:
this would only work out if there is an association on both models

as it stands, there is only a belongs_to association in the organization model but there is no corresponding association to organization in the country model